includes/api/class-wc-admin-rest-onboarding-profile-controller.php

```diff
@@ -283,15 +283,27 @@ public static function get_profile_properties() {
 					'other',
 				),
 			),
-			'theme'           => array(
+			'business_extensions' => array(
```

Comment:
We'll need a couple quick changes in  `tests/api/onboarding-profile.php` to get tests ✅

Reply:
Done.